File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/ApportoMacroelementi.ts
import { RegolamentoConcimazione } from './RegolamentoConcimazione';
import { FinalitaPianoConcimazione } from './FinalitaPianoConcimazione';
import { FaseCicloColturale } from "./fase";


export class ApportoMacroelementi {
    pianoConcimazione: RegolamentoConcimazione;
    tipologia: FinalitaPianoConcimazione;
    fase: FaseCicloColturale;
    n: number;
    p2o5: number;
    k2o: number;
    mgo: number;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/ApportoMacroelementi.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class ApportoMacroelementi
    {

        public RegolamentoConcimazione pianoConcimazione { get; set; }
        public FinalitaPianoConcimazione tipologia { get; set; }

        /// <summary>
        /// creare classe: capire da dove arriva per trovare nomenclaura per classe
        /// </summary>
        public FaseCicloColturale fase { get; set; }
        public double? n { get; set; }
        public double? p2o5 { get; set; }
        public double? k2o { get; set; }
        public double? mgo { get; set; }


        public ApportoMacroelementi()
        {
        }
    }
}
